mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -73,6 +74,7 @@ void SCFDialect::initialize() {
 #include "mlir/Dialect/SCF/IR/SCFOps.cpp.inc"
       >();
   addInterfaces<SCFInlinerInterface>();
+  declarePromisedInterface<ConvertToEmitCPatternInterface, SCFDialect>();
```

Comment:
Might want to move this down before `declarePromisedInterface<ValueBoundsOpInterface, ForOp>();`?